### ISSUE TITLE
Fix formatting for contract addresses in DSD

### ISSUE THREAD

**Issue by yanivtal** [graphprotocol/graph-node#149]
Right now addresses need to have quotes around them. This is awkward.
![image](https://user-images.githubusercontent.com/1707828/42347220-12c62f10-805a-11e8-8863-c60e2a9db59d.png)

Instead, addresses should be written as:
0x........ without quotes.

**Comment by davekaj** [comment]
@yanivtal looks like it slipped by, I'll take a look at it next week and see if I can make a small PR to fix this

**Comment by Jannis** [comment]
This is partially fixed by the `feature/block-stream` branch, which adds support for also parsing addresses that start with `0x`.

I'm fairly sure that `serde_yaml` assumes anything unquoted starting with `0x...` to be a number, so this will require more work.

**Comment by davekaj** [comment]
Seems like we have the 0x in, which is good! 

Sounds like removing the quotes might be more trouble than its worth, maybe we should just close this issue once #408 is merged ?

**Comment by davekaj** [comment]
I'm going to close this, barring any objections today

**Comment by Jannis** [comment]
The quotes are still necessary. Low priority.
